benchopt/utils/slurm_executor.py

```diff
@@ -22,7 +23,14 @@ def get_slurm_launch():
     return _LAUNCHING_SLURM
 
 
-def get_slurm_executor(slurm_config, timeout=100, job_name="benchopt_run"):
+def get_slurm_folder(benchmark):
```

Comment:
sorry after some bike shedding:

- could you put this as a method of the `Benchmark` class, with the constant define in `benchmark.py`?
- could you pass the benchmark to the function`get_slurn_executor` to get the folder with the method?

This way, the slurm folder will be the same, wherever the call to `benchopt run` is called .

Reply:
Sure !